includes/Renderer/Story/Image.php

```diff
@@ -81,11 +82,12 @@ public function render( array $args = [] ) {
 				<?php
 				if ( ! empty( $this->story->get_poster_portrait() ) ) {
 					printf(
-						'<img src="%1$s" width="%2$d" height="%3$d" alt="%4$s" />',
+						'<img src="%1$s" width="%2$d" height="%3$d" alt="%4$s" class="%5$s" />',
 						esc_url( $this->story->get_poster_portrait() ),
 						absint( $args['width'] ),
 						absint( $args['height'] ),
-						esc_attr( $this->story->get_title() )
+						esc_attr( $this->story->get_title() ),
+						esc_attr( 'wp-image-' . $this->story->get_thumbnail_id() )
```

Comment:
This attribute is required. See this block. 

https://github.com/WordPress/wordpress-develop/blob/e2feb7cf77603dcb0622b105377be7a37ffabf99/src/wp-includes/media.php#L1789-L1821